src/clang/misc.jl

```diff
@@ -0,0 +1,6 @@
+typealias FILE Void # not sure if this works
+
+# workaround error in common.jl
+# https://github.com/OSGeo/gdal/blob/d7451d90134f6ca4dea8551b3a5d8591d8e4efad/gdal/port/cpl_port.h#L512-L534
+# this is wrapped as: const CPL_LSBPTR16 = x
+x = nothing
```

Comment:
This doesn't seem right.. Haha. The definitions are over at https://github.com/OSGeo/gdal/blob/d7451d90134f6ca4dea8551b3a5d8591d8e4efad/gdal/port/cpl_port.h#L442-L491. Perhaps comment them out, or hardcode the structs?